modules/core/04-channel/keeper/upgrade.go

```diff
@@ -553,6 +553,16 @@ func (k Keeper) WriteUpgradeOpenChannel(ctx sdk.Context, portID, channelID strin
 		k.SetNextSequenceAck(ctx, portID, channelID, upgrade.NextSequenceSend)
 	}
 
+	// set the counterparty next sequence send in order to have upper bound to prune to
+	k.SetNextSequenceSend(ctx, portID, channelID, counterpartyUpgrade.NextSequenceSend)
```

Comment:
ref [aditya's comment in the other pr](https://github.com/cosmos/ibc-go/pull/5438#discussion_r1431604728), the test can be updated to make sure these are set (pruning and next seq send)

Reply:
This should `k.SetCounterpartyNextSequenceSend`, right?